docs/release-notes/NuGet-6.7.md

```diff
@@ -0,0 +1,175 @@
+---
+title: NuGet 6.7 Release Notes
+description: Release notes for NuGet 6.7 including new features, bug fixes, and DCRs.
+author: jeffkl
+ms.author: jeffkl
+ms.date: 7/31/2023
+ms.topic: conceptual
+---
+
+# NuGet 6.7 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.7**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.7](https://visualstudio.microsoft.com/downloads/) | [7.0.400](https://dotnet.microsoft.com/download/dotnet-core/7.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2022 with.NET Core workload
+
+## Summary: What's New in 6.7
+
+* NuGetAudit should warn when no vulnerability data available - [#12610](https://github.com/NuGet/Home/issues/12610)
+
+* remove period from NU190X - [#12600](https://github.com/NuGet/Home/issues/12600)
+
+* Package Source Mapping status for selected package in Details Pane - [#12586](https://github.com/NuGet/Home/issues/12586)
+
+* Add VulnerabilityInfo APIs into NuGet.Protocol - [#12518](https://github.com/NuGet/Home/issues/12518)
+
+* Signing:  raise actionable message on Linux if verification results in untrusted failure - [#12459](https://github.com/NuGet/Home/issues/12459)
+
+* Warn when vulnerabilities are detected during PackageReference restore  - [#12289](https://github.com/NuGet/Home/issues/12289)
+
+* Package Source Mapping UI - PM UI Actions support source pinning & create mappings in nuget.config - [#11797](https://github.com/NuGet/Home/issues/11797)
+
+* [Feature]: Show which package versions are vulnerable in the VS PMUI package details pane version dropdown - [#11127](https://github.com/NuGet/Home/issues/11127)
+
+* Suggestion:  NuGet config -list command to list configuration file locations - [#8420](https://github.com/NuGet/Home/issues/8420)
```

Comment:
Firstly, I believe it was only added to `dotnet nuget ...`, not to `nuget.exe`, so the name needs to be improved.

Secondly, wasn't this merged into a feature branch, but never merged into dev, and therefore isn't part of the release?
https://github.com/NuGet/NuGet.Client/tree/dev-v-dainiza-feature-dotnet-nuget-config/src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/ConfigCommands
https://github.com/NuGet/NuGet.Client/tree/release-6.7.x/src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/ConfigCommands

Reply:
I agree. We need to remove this. I've changed the milestone for that.